### ISSUE TITLE
Points with size 0 are not visible

### ISSUE THREAD

**Issue by domoritz** [vega/vega#1186]
See https://github.com/vega/vega-lite/issues/3488

**Comment by domoritz** [comment]
@jheer, is this on your radar?

**Comment by jheer** [comment]
Yes, but may not get to it for a week or so. Short version: size zero intuitively maps to "not visible"; moreover, this may go hand-in-hand within the underlying d3 shape generators, so may be largely outside of Vega itself. I need to take a closer look, but already I wonder if this should be treated as an issue of appropriately configuring scale ranges.

**Comment by jheer** [comment]
Also, whether strokes are visible or not over a span of zero is currently treated as a rendering-level concern, and can depend on the stroke line-cap type (butt, round, etc).

**Comment by domoritz** [comment]
If you decide that this is correct in Vega, what do you suggest we should generate in Vega-Lite? As a hack, I could imagine generating a scale range of `[1e-6, maxSize]`.
